tests/articulation_tests.ts

```diff
@@ -240,6 +241,15 @@ function verticalPlacement(options: TestOptions, contextBuilder: ContextBuilder)
   ];
 
   Formatter.FormatAndDraw(ctx, stave, notes);
+
+  // Render bounding boxes
+  if (options.params.bbox === true) {
```

Comment:
Same here. It could be written as:

 if (options.params.drawBoundingBox) {

Reply:
done